sdk/lib/rtl/priv.c

```diff
@@ -486,3 +486,78 @@ RtlAdjustPrivilege(IN ULONG Privilege,
 
     return STATUS_SUCCESS;
 }
+
+/*
```

Comment:
Maybe add a doxygen-type documentation? (since this function is undocumented on the internet, it better be documented here in our code).

Reply:
Of couse, I gonna fix :)